modules/logging-loki-log-access.adoc

```diff
@@ -74,16 +74,36 @@ subjects:
 // tag::CustomAdmin[]
 == Custom admin group access
 
-If you have a large deployment with a number of users who require broader permissions, you can create a custom group using the `adminGroup` field. Users who are members of any group specified in the `adminGroups` field of the `LokiStack` CR are considered admins. Admin users have access to all application logs in all namespaces, if they also get assigned the `cluster-logging-application-view` role.
+// tag::LokiMode[]
+If you have a large deployment with several users who require broader permissions, you can create a custom group using the `adminGroup` field. Users who are members of any group specified in the `adminGroups` field of the `LokiStack` CR are considered administrators. 
+// end::LokiMode[]
+
+// tag::NetObservMode[]
+If you need to see cluster-wide logs without necessarily being an administrator, or if you already have any group defined that you want to use here, you can specify a custom group using the `adminGroup` field. Users who are members of any group specified in the `adminGroups` field of the `LokiStack` CR have the same read access to logs as administrators. 
```

Comment:
Not fully sure how tags work, but if this is the first instance of CR in whichever section this is included in, then same comment as above:
```suggestion
If you need to see cluster-wide logs without necessarily being an administrator, or if you already have any group defined that you want to use here, you can specify a custom group using the `adminGroup` field. Users who are members of any group specified in the `adminGroups` field of the `LokiStack` custom resource (CR) have the same read access to logs as administrators. 
```

Reply:
Yeah I was gonna say, the other one you commented on is for the Loki docs, so I think this one is actually the one I'll need to change, since its the content I own.